pkg/kubectl/describe.go

```diff
@@ -953,7 +935,57 @@ func describeNode(node *api.Node, pods []*api.Pod, events *api.EventList) (strin
 	})
 }
 
-func filterNonRunningPods(pods []*api.Pod) []*api.Pod {
+func describeNodeResource(pods []*api.Pod, node *api.Node, out io.Writer) {
+	totalMilliCPUReq := int64(0)
+	totalMemoryReq := int64(0)
+	fractionPodCPUReq := float64(0)
+	fractionPodMemoryReq := float64(0)
+	fractionTotalCPUReq := float64(0)
+	fractionTotalMemoryReq := float64(0)
+	totalMilliCPULimit := int64(0)
+	totalMemoryLimit := int64(0)
+	fractionPodCPULimit := float64(0)
+	fractionPodMemoryLimit := float64(0)
+	fractionTotalCPULimit := float64(0)
+	fractionTotalMemoryLimit := float64(0)
+	for _, pod := range pods {
+		podTotalMilliCPUReq := int64(0)
+		podTotalMemoryReq := int64(0)
+		podTotalMilliCPULimit := int64(0)
+		podTotalMemoryLimit := int64(0)
+
+		for ix := range pod.Spec.Containers {
+			requests := pod.Spec.Containers[ix].Resources.Requests
+			podTotalMilliCPUReq += requests.Cpu().MilliValue()
+			podTotalMemoryReq += requests.Memory().Value()
+
+			limits := pod.Spec.Containers[ix].Resources.Limits
+			podTotalMilliCPULimit += limits.Cpu().MilliValue()
+			podTotalMemoryLimit += limits.Memory().Value()
+		}
+		totalMilliCPUReq += podTotalMilliCPUReq
+		totalMemoryReq += podTotalMemoryReq
+		totalMilliCPULimit += podTotalMilliCPULimit
+		totalMemoryLimit += podTotalMemoryLimit
+		fractionPodCPUReq = float64(podTotalMilliCPUReq) / float64(node.Status.Capacity.Cpu().MilliValue()) * 100
+		fractionPodMemoryReq = float64(podTotalMemoryReq) / float64(node.Status.Capacity.Memory().Value()) * 100
+		fractionPodCPULimit = float64(podTotalMilliCPULimit) / float64(node.Status.Capacity.Cpu().MilliValue()) * 100
+		fractionPodMemoryLimit = float64(podTotalMemoryLimit) / float64(node.Status.Capacity.Memory().Value()) * 100
+		fmt.Fprintf(out, "  %s\t%s\t\t%d (%d%% of total)\t\t%d (%d%% of total)\t\t%d (%d%% of total)\t\t%d (%d%% of total)\n", pod.Namespace, pod.Name, podTotalMilliCPUReq, int64(fractionPodCPUReq), podTotalMemoryReq, int64(fractionPodMemoryReq), podTotalMilliCPULimit, int64(fractionPodCPULimit), podTotalMemoryLimit, int64(fractionPodMemoryLimit))
```

Comment:
The sum of limits may exceed capacity. Showing that as a percentage of capacity will be confusing.